### ISSUE TITLE
Redundant inputs? 39 and 45

### ISSUE THREAD

**Issue by jennirinker** [DTUWindEnergy/BasicDTUController#33]
They both seem to specify the same thing. Overspeed percentage.

**Comment by ThanasisBarlas** [comment]
Wrong. The two constants serve different controller functions. One is supervisory function.

**Comment by jennirinker** [comment]
Here's my understanding of the controller. Let me know where I'm wrong.

Constant 39 becomes MoniVar%overspeed -- this will initial turbine shutdown if overspeed occurs, and the controller flag will be 1. (lines 393 through 398 in turbine_controller.f90).

Constant 45 becomes SafetySystemVar%overspeed -- this will initiate emergency turbine shutdown if overspeed occurs and also triggers a safety system printout to the console. (lines 29 through 32 in safety_system.f90)

What cases arise where we need to have two separate monitoring/shutdown sequences for overspeeds?

**Comment by ThanasisBarlas** [comment]
According to Morten's original idea, the turbine level monitoring deals with normal shut-down procedures while the supervisory level deals with emergency procedures.

The text in the report:
https://github.com/DTUWindEnergy/BasicDTUControllerReport/blob/4cbe3d42e3deedb4747da7eb24c7e8d081e3ff7c/description.tex#L342

If we think this is not necessary we should change it, but it's closer to a real controller implementation.
